src/data/actions/pingStudio.js

```diff
@@ -0,0 +1,19 @@
+export const PING_RESPONSE = 'PING_RESPONSE';
+
+function pingResponse(response) {
+  return {
+    type: PING_RESPONSE,
+    status: response.status,
+  };
+}
+
+export function pingStudio() {
```

Comment:
This is an asynchronous action creator, also known as a thunk. Action creator logic is polymorphic, so if it returns an object (like `pingStudio` above) it is treated as synchronous. However, if, like this one, it returns a function, it is treated as asynchronous. http://redux.js.org/docs/advanced/AsyncActions.html